lib/payment/merchant_stripe.rb

```diff
@@ -0,0 +1,184 @@
+module ShorePayment
+  #
+  class StripeHash
+    def initialize(attrs = {})
+      update_attributes(attrs)
+    end
+
+    # Update object with this method. We have to take care updating 'nested'
+    #   objects
+    def update_attributes(attrs = {})
+      attrs.each_pair do |attr, value|
+        send(:"#{attr}=", value) if respond_to?(:"#{attr}=")
+      end if attrs
+    end
+  end
+
+  # Conversion between day of birth's Hash and Date representation
+  module StripeDobDate
+    def dob_date
+      if @dob && dob_present?
+        Date.new(@dob.year.to_i, @dob.month.to_i, @dob.day.to_i)
+      end
+    end
+
+    def dob_date=(new_date)
+      date = new_date.to_date if new_date.respond_to?(:to_date)
+      @dob = DateOfBirth.new(
+        year: date.year,
+        month: date.month,
+        day: date.day
+      ) if date
+    end
+
+    private
+
+    def dob_present?
+      @dob.year.present? && @dob.month.present? && @dob.day.present?
+    end
+  end
+
+  # Representation of a {DOB} object in the Organization response
+  class DateOfBirth < StripeHash
+    attr_accessor :day, :month, :year
+  end
+
+  # Representation of an {Address} object in the Organization response
+  class Address < StripeHash
+    attr_accessor :city, :country, :line1, :line2, :postal_code, :state
+  end
+
+  # Representation of an {AdditionalOwner} object in the Organization response
+  class AdditionalOwner < StripeHash
+    include StripeDobDate
```

Comment:
Maybe StripeDobDate is not a good name. ```include``` means 'behave like'. So what you are saying is 'behaves like a StripeDobDate'. Maybe